sdk/tests/conformance2/transform_feedback/transform_feedback.html

```diff
@@ -204,6 +209,46 @@
 
     tf = null;
     program = null;
+
+    // Check the result of the query. It should not be available yet.
+    var numEarlyTests = 50000;
```

Comment:
Seems a bit excessive, but Ken gave a compelling explanation offline. Might help to have a comment explaining as much.